### PR TITLE
Deployment duration metric

### PR DESCRIPTION
**Pull request by mpolden**
Last commit is the interesting one.

### CODE REVIEW

**Comment by mpolden** [comment]
@bratseth ping

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/MetricsReporter.java [hunk 1]
```diff
@@ -121,8 +129,37 @@ private double deploymentFailRatio() {
...
+                              .collect(Collectors.toMap(Application::id,
+                                                        application -> averageDeploymentDuration(application, now)));
+    }
+
+    private Duration averageDeploymentDuration(Application application, Instant now) {
+        List<Duration> jobDurations = application.deploymentJobs().jobStatus().values().stream()
+                                                 .filter(status -> status.lastTriggered().isPresent())
+                                                 .map(status -> {
+                                                     Instant triggeredAt = status.lastTriggered().get().at();
+                                                     Instant completedAt = status.lastCompleted()
```
Comment (bratseth) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/MetricsReporter.java]: "completedAt" is misleading; call this "runningUntil" or something?

**Review by bratseth — APPROVED** [review]
Great. And sorry for missing this on Friday.

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/MetricsReporter.java (hunk 1 above)
Comment (mpolden) [reply controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/MetricsReporter.java]: Done.

**Pull request merged**